ynr/apps/people/models.py

```diff
@@ -678,3 +694,16 @@ def get_display_image_url(self):
 
     def __str__(self):
         return self.name
+
+    @property
+    def edits_allowed(self):
```

Comment:
I know at the moment, editing is either enabled or disabled globally, but we also talked about limiting this by group in future. Is it worth thinking of the method signature here as `person.user_can_edit_person(self, user)` (with a user param which currently happens to be ignored) in the same way we've got `ballot.user_can_edit_membership(self, user)` here: https://github.com/DemocracyClub/yournextrepresentative/blob/516ee02bd5ea510b460ca902f27c397890439f50/ynr/apps/candidates/models/popolo_extra.py#L191 ?